pkg/agent/flowexporter/connections/conntrack_connections.go

```diff
@@ -71,11 +81,23 @@ func NewConntrackConnectionStore(
 		connectionStore:       NewConnectionStore(npQuerier, podStore, proxier, o),
 		connectUplinkToBridge: o.ConnectUplinkToBridge,
 		l7EventMapGetter:      l7EventMapGetterFunc,
+		networkPolicyWait:     networkPolicyWait,
 	}
 }
 
 // Run enables the periodical polling of conntrack connections at a given flowPollInterval.
 func (cs *ConntrackConnectionStore) Run(stopCh <-chan struct{}) {
+	if cs.networkPolicyWait != nil {
+		klog.Info("Waiting for NetworkPolicies to become ready")
+		if err := cs.networkPolicyWait.WaitUntil(stopCh); err != nil {
```

Comment:
```suggestion
		// Wait for networkPolicyController to either become ready or cancelled via stopCh
		if err := cs.networkPolicyWait.WaitUntil(stopCh); err != nil {
```

The `WaitUntil` function at first glance seemed like it would wait until something came in on `stopCh`. Not a change we need to make in this PR. Not sure what better name to use though.